pom.xml

```diff
@@ -245,11 +245,24 @@
             <scope>test</scope>
         </dependency>
 
+        <dependency>
+            <groupId>org.springframework</groupId>
+            <artifactId>spring-test</artifactId>
+            <version>4.2.1.RELEASE</version>
+            <scope>test</scope>
+        </dependency>
+
         <!-- Testing JSON -->
         <dependency>
             <groupId>com.jayway.restassured</groupId>
             <artifactId>json-path</artifactId>
-            <version>2.5.0</version>
+            <version>2.8.0</version>
+            <scope>test</scope>
+        </dependency>
+        <dependency>
+            <groupId>com.jayway.jsonpath</groupId>
+            <artifactId>json-path</artifactId>
```

Comment:
Oh, I see. The other json-path has been introduced when you merged the connector / scim-schema into the server. It had been used for the de-/serialization tests in the scim-schema. So I think one is enough. Can rewrite the tests to use the one provided by rest-assured. At least, this is lib for tests. I think the other one is more suited for production.